src/osd/OSD.cc

```diff
@@ -1388,21 +1388,13 @@ MOSDMap *OSDService::build_incremental_map_msg(epoch_t since, epoch_t to,
   if (since < m->cluster_osdmap_trim_lower_bound) {
```

Comment:
is 'max' used at all now?

Reply:
Yes

Reply:
Probably missed something, but I did not see it used in the code